webapp/src/shared/components/Input/Input.css

```
@@ -0,0 +1,39 @@
+.form-container {
+  width: 312px;
+}
+
+.input-container {
+  margin-top: 0.3rem;
+  display: flex;
```

```suggestion
  display: flex;
  align-items: center;
```
suggestion: This centers the icon and the input horizontally inside the input container.